src/bridge/rnastructure.cpp

```diff
@@ -93,57 +131,46 @@ std::vector<subopt::SuboptResult> RNAstructure::SuboptimalIntoVector(
   // Arguments: structure, data tables, percentage delta, absolute delta, nullptr, nullptr, false
   verify(int16_t(delta) == delta, "delta too big");
   alltrace(structure.get(), data_.get(), 100, int16_t(delta), nullptr, nullptr, false);
-  auto s_list = StructureToSecondarys(*structure);
-  std::vector<subopt::SuboptResult> res;
-  for (int i = 0; i < static_cast<int>(s_list.size()); ++i) {
-    // TODO: Convert CTDs?
-    res.emplace_back(subopt::SuboptResult(
-        Energy(structure->GetEnergy(i + 1)), tb::TracebackResult(std::move(s_list[i]), Ctds())));
-  }
-  return res;
+  return StructureToSuboptVector(*structure);
 }
 
 part::PartResult RNAstructure::Partition(const Primary& r) const {
   const auto structure = LoadStructure(r);
+  auto pf_state = CallPartitionFunction(structure.get());
```

Comment:
Here and other locations, could you call this just state?

Reply:
Done.

FTR I consider pr_state more readable and reasonably inline with convention (i.e. s_list, pfdata). Can you explain the reason "state" is preferred?